### PR TITLE
Feature/hw 6

### PR DESCRIPTION
**Pull request by SlavikSurminskiy**
- Page for creating new course
- Page for editing existing course

### CODE REVIEW

File: src/app/services/courses/courses.service.ts [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
+
+  getCourse(id: string): ICourse | undefined {
+    return this._courses.find((course) => course.id === id);
+  }
+
+  updateCourse(id: string, data: CourseUpdate): ICourse[] {
+    const target = this._courses.find((course) => course.id === id);
+    Object.assign(target, data);
+
+    return this._courses;
```
Comment (roksolanachipko) [inline src/app/services/courses/courses.service.ts]: Please delete unneeded return

File: src/app/services/courses/courses.service.ts [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+  }
+
+  deteteCourse(id: string): ICourse[] {
+    this._courses = this._courses.filter((course) => course.id !== id);
+
+    return this._courses;
```
Comment (roksolanachipko) [inline src/app/services/courses/courses.service.ts]: Please delete unneeded return

**Review by roksolanachipko — APPROVED** [review]

File: src/app/services/courses/courses.service.ts (hunk 1 above)
Comment (SlavikSurminskiy) [reply src/app/services/courses/courses.service.ts]: Done

File: src/app/services/courses/courses.service.ts (hunk 2 above)
Comment (SlavikSurminskiy) [reply src/app/services/courses/courses.service.ts]: Done

**Pull request merged**